infra/modules/aws-psoxy-lambda/variables.tf

```diff
@@ -79,3 +74,18 @@ variable "log_retention_in_days" {
   description = "number of days to retain logs in CloudWatch for this psoxy instance"
   default     = 7
 }
+
+variable "global_parameters" {
```

Comment:
if it's arns, I'd just call it `global_parameter_arns` to be clear on the expected type.